frontend/src/app/components/MobileDialog/index.js

```diff
@@ -82,12 +82,23 @@ export default class MobileDialog extends React.Component {
 
     const headerImg = ios_url ? (<a href={ios_url} target="_blank"><img className="mobile-header-img" src="/static/images/ios-light.svg"/></a>) : (<a href={android_url} target="_blank"><img className="mobile-header-img" src="/static/images/google-play.png"/></a>);

```

Comment:
Bonus nit: Add `rel="noopener noreferrer"` to both the `target="_blank"` links.

Reply:
Man, this ain't even in my PR

Reply:
I'll fix this in my metrics modal pr, since I have to fix conflicts there anyways